tests/db/test_bigchain_api.py

```diff
@@ -1822,6 +1856,8 @@ def test_get_subcondition_from_vk(self, b, user_sk, user_vk):
             subcondition = condition.get_subcondition_from_vk(owner_after)[0]
             assert subcondition.public_key.to_ascii().decode() == owner_after
 
+    @pytest.mark.skipif(reason='@Dimi need explanation of what is actually'
+                        'going on here')
     @pytest.mark.usefixtures('inputs')
```

Comment:
These are the escrow tests. Basically they are hold accounts that can either execute or roll back.
The expiry time defines which one of both is valid at a given point in time.

asset flow: `[A] -> [A, B] -> [A] if expired else [B]`